packages/charts/src/components/legend/types.ts

```diff
@@ -0,0 +1,8 @@
+export type LegendCardSettings = {
+  backgroundColor?: string;
+  border?: string;
```

Comment:
is it still valid? 

` borderWidth?: number;  borderColor?: string;`

Reply:
Yes, it is valid. Thats why we decided to separate legend card settings type from regular card settings type, to not break compatibility with legend styling.